### PR TITLE
Fix gateway remote deployment

### PR DESCRIPTION
**Pull request by Tanguyabel**
---

- [x] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: now/executor/gateway/now_gateway.py [hunk 1]
```diff
@@ -108,6 +108,10 @@ def __init__(self, user_input_dict: Dict = {}, **kwargs):
         with open(os.path.join(os.path.expanduser('~'), 'user_input.json'), 'w') as f:
             json.dump(self.user_input.__dict__, f)
 
+        # remove potential clashing arguments from kwargs
+        kwargs.pop("port")
```
Comment (violenil) [inline now/executor/gateway/now_gateway.py:112]:
```suggestion
        kwargs.pop("port", None)
```

File: now/executor/gateway/now_gateway.py [hunk 2]
```diff
@@ -108,6 +108,10 @@ def __init__(self, user_input_dict: Dict = {}, **kwargs):
...
+        kwargs.pop("protocol")
```
Comment (violenil) [inline now/executor/gateway/now_gateway.py:113]:
```suggestion
        kwargs.pop("protocol", None)
```

**Review by joschkabraun — APPROVED** [review]
After Isabelle's suggestion have been merged, LGTM

File: now/executor/gateway/now_gateway.py (hunk 1 above)
Comment (Tanguyabel) [reply now/executor/gateway/now_gateway.py:112]: indeed, without a default value passed, this call fails if the key does not exist. Fixing now!

**Review by violenil — APPROVED** [review]
LGTM!

**Pull request merged**